test/one/profiler/test/Test.java

```diff
@@ -36,6 +36,8 @@
 
     Os[] os() default {};
 
+    int cpu() default -1;
```

Comment:
Since the feature is used in one test only (and it's unlikely to be used anywhere else in the tests), may be it's not worth adding as a generic feature of the test framework. Perhaps you can just create an utility method in `CpuTests` to run `taskset`.

Reply:
c290e6f189a21265a019a197d4cc852fd3f68017